src/context/GeoStylerContext/GeoStylerContext.tsx

```diff
@@ -84,7 +82,26 @@ export const useGeoStylerLocale = <T extends keyof GeoStylerLocale>(key: T): Geo
   return structuredClone(ctx.locale[key]);
 };
 
-export const useGeoStylerUnsupportedProperties = (): any => {
+export const useGeoStylerUnsupportedProperties = <T extends Symbolizer>(symbolizer: T) => {
   const ctx = useContext(GeoStylerContext);
-  return structuredClone(ctx.unsupportedProperties);
+
+  if (!ctx.unsupportedProperties) {
+    return {
+      getSupportProps: (propName: keyof T) => {}
```

Comment:
Should this have the same structure as the return value at the end of the function? I.e.

```js
return {
    {},
    getSupportProps: (propName: keyof T) => {}
  };
```